dockstore-client/src/main/java/io/dockstore/client/cli/nested/WorkflowClient.java

```diff
@@ -303,11 +303,11 @@ private Workflow getDockstoreWorkflowByPath(String path) {
         // simply getting published descriptors does not require permissions
         Workflow workflow = null;
         try {
-            workflow = workflowsApi.getPublishedWorkflowByPath(path, null, false, null);
+            workflow = workflowsApi.getPublishedWorkflowByPath(path, "versions", false, null);
```

Comment:
This means that the 1.10 client will break against the 1.11 web service, doesn't it? I don't think there's much to do about it at this point.

Reply:
Yes